site/static/js/main.js

```diff
@@ -8,8 +8,18 @@ $(document).ready(function(){
         $(this).toggleClass('fa-plus-square-o fa-minus-square-o');
     });
 
+    let notifFlagExist = document.cookie.split(';').filter(item => { return item.indexOf('notifFlag=') >= 0}).length
+
+    if(notifFlagExist){
+        $(".notification-bar").addClass("closed");
+        $("header").removeClass("with-notification-bar");
+    }
+
     $('.notification-bar__close').on('click', function(){
-       $(".notification-bar").addClass("closed");
-       $("header").removeClass("with-notification-bar");
+        if(!notifFlagExist){
+            $(".notification-bar").addClass("closed");
+            $("header").removeClass("with-notification-bar");
+        }
+        document.cookie = 'notifFlag=true'
```

Comment:
Nit pick on namin: Should this be `hideBanner` instead?

Reply:
Done